package.json

```diff
@@ -18,24 +18,36 @@
   },
   "devDependencies": {
     "@babel/cli": "^7.20.7",
-    "@babel/core": "^7.7.2",
+    "@babel/core": "^7.20.12",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@babel/preset-env": "^7.7.1",
+    "@babel/preset-typescript": "^7.18.6",
+    "@types/jest": "^29.4.0",
+    "@types/node": "^18.13.0",
```

Comment:
This `package.json` specifies node >= 16 as valid node versions. Shouldn't `@types/node` be pinned at 16 to maintain compatibility? At least I think the versions align. Also, should this even a required package for a lib intended to run on a browser?

Reply:
> Also, should this even a required package for a lib intended to run on a browser?

Yeah. it's needed for compiling one of the dependencies unfortunately. 

> This package.json specifies node >= 16 as valid node versions. Shouldn't @types/node be pinned at 16 to maintain compatibility?

Good question. Since it could be greater than 16, don't we need to also support 18 version?

Reply:
Since it works as is in a v16 environment I think this should be ok for now. but definitely worth keeping an eye on.